assets/scss/stripe.scss

```diff
@@ -1,5 +1,123 @@
-@include responsiveBase() {
-	.stripe{
+/*doc
+---
+title: Stripe
+---
+A `.stripe` class is rendered by the `<Stripe>` component to
+visually divide the page with borders or stripes of color.
+
+To align content within a stripe, nest children in a
+`<Bounds>` component.
+
+#### Stripe variants
+A class of `.stripe` is default; the following classes are
+optional variants:
```

Comment:
This was a temporary way to bring the necessary styles over to get the components into use. I showed Adam what we were doing with component styles when we started the `responsify` branch, and he was interested in keeping styles close to components.
This was a while ago, so I don't recall how he wants to handle this for the future of SQ2.